.github/workflows/pull-request-checks.yaml

```diff
@@ -44,8 +44,8 @@ jobs:
         run: make -C src -j2 CXX="ccache g++"
       - name: Run the ebmc tests with SAT
         run: make -C regression/ebmc test
-      - name: Run the ebmc tests with Z3
-        run: make -C regression/ebmc test-z3
+      #- name: Run the ebmc tests with Z3
+      #  run: make -C regression/ebmc test-z3
```

Comment:
Should these really be disabled?

Reply:
This needs https://github.com/diffblue/cbmc/pull/8123.
I won't wait for that, will merge #315.